docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Consider adding a link to the other notebook. Is it expected that readers have read the previous notebook first? Then consider mentioning it here and getting rid of all the content that better fits in the first notebook (like below "Note: JAX and Flax, like numpy are row-based systems, ...")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>That's a good question in fact. My assumption when writing this was that some folks might directly start with the Flax part, but that might not be the right approach. We probably need some balance between self contained and redundancy. @avital any opinion ?</p>

Reply:
<p>About the note, I've put it there because for me I was surprised to see that the kernel was the transpose of what I expected in the params, so I still think it's worth reminding people here.</p>